src/CodeFormatter.cs

```diff
@@ -157,6 +157,23 @@ private static async Task<(Solution solution, int filesFormatted)> FormatFilesIn
                     continue;
                 }
 
+                if (filesToFormat != null)
+                {
+                    var fileInArgumentList = false;

```

Comment:
We should use a case insensitive comparison for the file names. Also, we would like to end the search after we've found a match.

```csharp
var fileInArgumentList = filesToFormat.Any(path => document.FilePath.EndsWith(path, StringComparison.OrdinalIgnoreCase));
```

Reply:
What do you mean end the search when we find a match? Are you suggesting adding `break;` to the loop, which I guess `Any` takes care of in a sense?

Reply:
Done!